### PR TITLE
Xylix improvements

### PR DESCRIPTION
**Pull request by xylix**

### CODE REVIEW

File: backend/core/endpoints/channels.py [hunk 1]
```diff
@@ -64,11 +64,9 @@ def create_channel(
         if server is None:
             raise HTTPException(status_code=404, detail="Server not found")
         channel = schemas.ChannelCreate(
```
Comment (xylix) [inline backend/core/endpoints/channels.py:66]: Probably should discuss if we want to change this everywhere or leave it the old way

Comment (xylix) [inline backend/core/endpoints/channels.py:66]: Probably should discuss if we want to change this everywhere or leave it the old way

File: backend/core/endpoints/channels.py [hunk 2]
```diff
@@ -64,11 +64,9 @@ def create_channel(
...
-            **{
-                "discordId": channel.discord_id,
-                "name": channel_data.get("name"),
```
Comment (xylix) [inline backend/core/endpoints/channels.py:69]: Probably should discuss if we want to change this everywhere or leave it the old way

Comment (Natsku123) [reply backend/core/endpoints/channels.py:69]: Yea, we probably should change them all into this, there is no real reason to keep it the old way

**Review by Natsku123 — APPROVED** [review]

File: backend/core/endpoints/channels.py (hunk 2 above)
Comment (xylix) [reply backend/core/endpoints/channels.py:69]: Made the rest of the changes in acb143eeae13faa9106399177b0515a4e70c8e5a, now branch is ready to merge imo

**Review by Natsku123 — APPROVED** [review]

**Pull request merged**